### PR TITLE
Test Rule doc link

### PR DESCRIPTION
**Pull request by saif-ericsson**
Docs moved to github wiki and link created in front-end for Test Rule

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -23,12 +23,13 @@ ei.useSecureHttpBackend=false
 ei.backendInstancesPath=
 
 ###### EI Documentation Link Url ##########
-ei.eiffelDocumentationUrls={ "EI Frontend GitHub": "https://github.com/Ericsson/eiffel-intelligence-frontend",\
-                             "EI Backend GitHub": "https://github.com/Ericsson/eiffel-intelligence",\
+ei.eiffelDocumentationUrls={ "EI Frontend GitHub": "https://github.com/eiffel-community/eiffel-intelligence-frontend",\
+                             "EI Backend GitHub": "https://github.com/eiffel-community/eiffel-intelligence",\
                              "Eiffel Github": "https://github.com/Ericsson/eiffel",\
-                             "Eiffel Github main page": "http://ericsson.github.io/eiffel" }
+                             "Eiffel Github main page": "https://github.com/eiffel-community/eiffel",\
```
Comment (tobiasake) [inline src/main/resources/application.properties]: Remove "https://github.com/Ericsson/eiffel" link from this list.

Comment (saif-ericsson) [reply src/main/resources/application.properties]: link removed

**Comment by tobiasake** [comment]
Shouldn't you remove the test rules documentation from the source code also? Since it was(will be?) moved to wiki documentation pages.

**Comment by saif-ericsson** [comment]
test rule docs are already removed from the source code...

**Comment by tobiasake** [comment]
Ok,, it was hidden by show diff button/section in the files review view.
But I see now that you have removed the file: src/main/resources/templates/jmesPathRulesSetUp.html

**Pull request merged**